libraries/botbuilder-dialogs-adaptive/package.json

```diff
@@ -35,6 +35,7 @@
     "botbuilder-dialogs-adaptive-runtime-core": "4.1.6",
     "botbuilder-dialogs-declarative": "4.1.6",
     "botbuilder-lg": "4.1.6",
+    "botframework-connector": "4.1.6",
```

Comment:
Thought I only saw this in the tests?

Reply:
[Used to be](https://github.com/microsoft/botbuilder-js/pull/3792/files#diff-e3423f1173c51c9f256335a75a2705c495a2b163cbbb1291925708cbda444d26R5)